testdata/dnn/download_models.py

```diff
@@ -82,7 +83,29 @@ def get(self):
 
         print(' done')
         print(' file {}'.format(self.filename))
-        return self.verify()
+        candidate_verify = self.verify()
+        if not candidate_verify:
+            if os.path.exists(self.filename):
```

Comment:
It would be better to extract this block of code into a new class method, e.g. `handle_bad_download`:
```.py
if not self.verify():
    self.handle_bad_download();
    return False

return True
```